### ISSUE TITLE
/apollo/modules/storytelling/dag/storytelling.dag 占用cpu过高

### ISSUE THREAD

**Issue by looles** [ApolloAuto/apollo#15004]
![图片](https://github.com/ApolloAuto/apollo/assets/44311409/ff14dc54-764c-43c7-ad11-348d30ca10e8)
你好，我想问一下为什么apollo占用cpu100%
![图片](https://github.com/ApolloAuto/apollo/assets/44311409/67588912-3d1f-4c9c-9e93-ec649e93652a)
这是正常的吗？

**Comment by daohu527** [comment]
No, this is not normal. storytelling is single-threaded and all your CPUs is fully loaded, so you need to find out what is taking up those cpus.
